src/components/LoansByCategory/MFIRecommendations/MfiLoansWrapper.vue

```diff
@@ -48,28 +48,14 @@ export default {
 
 </script>
 
-<style lang="scss" scoped>
-	@import 'settings';
-	@include breakpoint(xxlarge) {
-		#carousel_exp {
-			position: relative;
-		}
-
-		#carousel_exp >>> section > div:nth-child(2) {
-			position: absolute;
-			top: -55px;
-			margin-left: -40px;
-		}
+<style scoped>
 
-		#carousel_exp >>> section > div:nth-child(2) > button {
-			width: 24px;
-			height: 24px;
-			margin-right: 15px;
-		}
+	#carousel_exp >>> .kv-carousel__controls {
```

Comment:
this component is in LBC page too, but working under an experiment setup that will be removed from this page including the mfi feat, so, this will be in lending home page only